lib/include/jxl/types.h

```diff
@@ -109,6 +109,10 @@ typedef struct {
   size_t align;
 } JxlPixelFormat;
 
+/** Data type holding the 4-character type name of an ISOBMFF box.
+ */
+typedef char JxlBoxType[4];
```

Comment:
I think it would be useful to add some constants/defines for some of the boxes, maybe for BROB, just `{'b', 'r', 'o', 'b'}`, but we can do this later.

Reply:
I agree that we can do it later